### PR TITLE
Adds a build-time table construction

### PR DESCRIPTION
**Pull request by SamHPurp**
This is a balance change; therefore I'd appreciate reasoned arguments for and against in the comments.

------

Currently you can instantly build tables. With no wait, whatsoever.

The issue with this is, it is a 'get-out-of-chases-free' card through maintenance, just by carrying table parts in your backpack. This gets you a spare 5+ seconds MINIMUM against a sec officer chaser... and if its a Sec-borg? They cannot do anything about the table.

By applying this delay, this weakens power-gamers, but keeps it as a tactic.

TL;DR you aren't pro-IKEA, you can't insta-build tables.

:cl: Purpose2
tweak: Adds a build-time on Tables & Racks.
/ :cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Racks?

**Comment by Spacemanspark** [comment]
Racks can be destroyed in a few hits.

**Comment by FlattestGuitar** [comment]
God, yes. This is stupidly powerful in the right hands. There's pretty much no way for security to counter a person using these.

**Comment by SamHPurp** [comment]
Those with Swedish disability get to ignore the build timer? 🤣

**Comment by Birdtalon** [comment]
May I ask how long the timer is?

**Comment by Fruerlund** [comment]
Awesome one of those things that has never been "looked" at.

**Comment by SamHPurp** [comment]
@Birdtalon yes, 2 seconds.

**Comment by taukausanake** [comment]
I think three seconds would be nice. What is the timer for most constructions/deconstructions?

**Comment by Anticept** [comment]
Those who are swede should also be able to cook the best meatballs.
